src/main/java/uk/gov/pay/directdebit/payments/dao/mapper/TransactionMapper.java

```diff
@@ -10,31 +10,35 @@
 import java.sql.SQLException;
 
 public class TransactionMapper implements ResultSetMapper<Transaction> {
-    private static final String ID_COLUMN = "id";
-    private static final String AMOUNT_COLUMN = "amount";
-    private static final String TYPE_COLUMN = "type";
-    private static final String STATE_COLUMN = "state";
+
+    private static final String TRANSACTION_ID_COLUMN = "transaction_id";
+    private static final String TRANSACTION_AMOUNT_COLUMN = "transaction_amount";
```

Comment:
Good job! I think I actually had a todo somewhere saying that we were overriding transaction and payment request amounts because we were not explicit as to which one we are picking 👍

Reply:
Actually, the todo was `//todo refactor transaction to contain a PaymentRequest`. One improvement we can make to the code is to pass a payment request object directly in Transaction, instead of having various payment requests fields (ie. external id, etc.) there. But we can do it in another pr, no probs

Reply:
Yes, will do the refactoring after selfservice integration, thanks